packages/cli/src/args.ts

```diff
@@ -0,0 +1,81 @@
+import chalk from 'chalk';
```

Comment:
This is mostly the original `args.ts` file from the `commands/deploy` directory, with the deploy-specific stuff left in that original place.